src/EffectHostInterface.h

```diff
@@ -49,12 +50,15 @@ class AUDACITY_DLL_API EffectHostInterface
 class wxDialog;
 class wxWindow;
 class EffectUIClientInterface;
-
+class EffectSettingsAccess;
+using EffectSettingsAccessPtr = std::shared_ptr<EffectSettingsAccess>;
```

Comment:
Suggest moving this alias to `EffectUIHostInterface` scope

Reply:
Good idea